### ISSUE TITLE
Fishoids always move off to the lower corner of the screen where they can't be shot.

### ISSUE THREAD

**Issue by bakerhillpins** [jessaskey/mhavocpe#203]
0.85 PE Demo Mode play.

I'm playing level 13.  Kill a few of the fishoids down to 1 or 2. Then let those fishoids come to the bottom where they circle to the Jaws theme. If you just leave them alone they will move off to the lower right corner of the screen.  This happens pretty quickly. Once they are there the catastrofigher can't reach them and you must sit and wait for the level bonus to count to zero. When it does you will be allowed to land.

**Comment by jessaskey** [comment]
This ended up being a bug from #104 which was an earlier change request, seems #104 was fine on Level 1 and 5 but not 13, 17, etc due to some difficulty settings. In #104 ended up leaving the fishoids with this 'double bounce' effect which was simply something I ended with because I wasn't able to get the fishoids to sucessfully 'go back up' and then re-attack. Well, I spent some time getting this finally ironed out and now the behavior is this... once a fishoid comes down, it will circle 4 time... on the 4th circle, it will then turn 'aqua' again (not agressive) and it will fly back up towards the station... at about mid-screen, it will turn red again and then come back down towards the player and repeat this action until killed. I will test this a little more but I believe this is good now.
